config/routes.rb

```diff
@@ -0,0 +1,7 @@
+Rails.application.routes.draw do
+  root "tasks#index"
+  resources :tasks
+
+  patch "/tasks/:id", to: "tasks#update"
```

Comment:
You shouldn't need to include the `update` route separately from `resources` here.